blockbase/sass/base/_alignment.scss

```diff
@@ -47,8 +47,12 @@
 // This was added for the 'site-logo' block which centers with an 'align:center' attribute
 // instead of 'textAlign' center which sets an .aligncenter class instead of a has-text-align-center
 // class which would do this for us.  I'm not sure why but this centers things appropriately.
+// Display and margin properties added to support image alignment from the classic editor.
 .aligncenter {
 	text-align: center;
+    display: block;
```

Comment:
Nitpick: This looks like spaces not tabs

Reply:
Ha yes it definitely does! I'll change it 😁

Reply:
Are there any cases where adding `display: block` to an `aligncenter` block would result in unwanted behavior? 

It's pretty low specificity so I think it's okay, but it gave me pause.

Reply:
> Are there any cases where adding display: block to an aligncenter block would result in unwanted behavior?

Yeah this is definitely valid. I guess the possible issue would be that content would be stacked that you may have intended to be inline, from applying this class? But I can't think of an example where this would happen when centering (i.e. you'd probably expect things to stack and center, and only expect left/right alignment to be inline, I think..)